content/4_Gold/Meet_In_The_Middle.mdx

```diff
@@ -0,0 +1,220 @@
+---
+id: meet-in-the-middle
+title: 'Meet In The Middle'
```

Comment:
this has only appeared in silver, why put it in Gold?

Reply:
i don't think this topic really fits in server because it's kind of misc

Reply:
then why put it in gold? :P

Reply:
it's important enough to be prone to appear again, most likely in a higher division since that silver contest only had a 650 cutoff